src/main.cpp

```diff
@@ -49,11 +52,22 @@ void newRange()
     Serial.print("\t RX power: ");
     Serial.print(DW1000Ranging.getDistantDevice()->getRXPower());
     Serial.println(" dBm");
+
+    if ((DW1000Ranging.getDistantDevice()->getRange() < 2.0))
```

Comment:
lets create 

```c
#define RANGE_THRESHOLD_METERS 2.0f
```

and use it here. Easier to tweak it later.

Reply:
:facepalm: